docs/en/appium-setup/parallel_tests.md

```diff
@@ -22,6 +22,8 @@ As long as your Appium and Appium bootstrap ports are between 0 and 65536, all t
 
 If you are using chromedriver or selendroid, set a different port for each server.
 
+If you are using [appium-uiautomator2-server](https://github.com/appium/appium-uiautomator2-server), set a different system port with `systemPort` capability since sometimes the same ports is conflictted such as [this issue](https://github.com/appium/appium/issues/7745).
```

Comment:
1. I would link to the uiautomator2 driver (not the server) since what is used in appium is first and foremost the driver.
2. `set a different system port` => `set a different system port for each Appium instance`
3. `sometimes the same port is conflictted such as` => `sometimes there can be a port conflict if different ports aren't used, such as in`